platforms/r3-corda-ent/charts/zone/values.yaml

```diff
@@ -0,0 +1,204 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# Default values for CENM Zone service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+#############################################################
+#                   Basic Configuration                     #
+#############################################################
+# This section contains the Corda Enterprise Zone metadata.
+metadata:
+  # Provide the namespace for the Corda Enterprise Zone.
+  # Eg. namespace: cenm
+  namespace:
+  # Provide the name of the deployment
+  # Eg. nodeName: zone
+  nodeName: 
+  # Provide prefix for deployment
+  # Eg. prefix: cenm
+  prefix:
+
+# Provide information regarding the Docker images used.
+image:
+  # Provide the alpine utils image, which is used for all init-containers of deployments/jobs.
+  # NOTE: The alpine image used is the base alpine image with CURL installed.
+  # Eg. initContainerName: hyperledgerlabs/alpine-utils:1.0
+  initContainerName:
+  # Provide the image for the main Zone container.
+  # Eg. zoneContainerName: corda/enterprise-zone:1.5.1-zulu-openjdk8u242
+  zoneContainerName:
+  # Name of Kubernetes secret storing credentials to Docker registry
+  # Provide the docker-registry secret created and stored in kubernetes cluster as a secret.
+  imagePullSecrets:
+    # Eg. name: regcred
+    - name:
+  # Pull policy to be used for the Docker image
+  # Eg. pullPolicy: Always
+  pullPolicy:
+
+#############################################################
+###                     Settings                          ###
+#############################################################
+config:
+  # Provide volume related specifications
+  volume:
+    # Eg. baseDir: /opt/cenm
+    baseDir:
+
+  pvc:
+    # Volume size for h2/ directory
+    # Eg. volumeSizeZoneH2: 1Gi
+    volumeSizeZoneH2:
+    # Volume size for logs/ directory
+    # Eg. volumeSizeZoneLogs: 5Gi
+    volumeSizeZoneLogs:
+    # Volume size for data/ directory
+    # Eg. volumeSizeZoneData: 1Gi
+    volumeSizeZoneData:
+
+  pod:
+    # Set memory limits of pod  
+    resources:
+      limits:
+        # Provide the limit memory for node
+        # Eg. memory: 1Gi
+        memory:
+      requests:
+        # Provide the requests memory for node
+        # Eg. memory: 1Gi
+        memory:
+
+  zoneJar:
+    # The directory where the Zone Service .jar file is stored
+    # Eg. path: bin
+    path:
+
+  podSecurityContext:
```

Comment:
These can be removed and hard coded straight in charts

Reply:
fixed